packages-exp/auth-exp/src/model/auth.d.ts

```diff
@@ -34,10 +43,22 @@ export interface Auth {
   currentUser: User | null;
   readonly name: AppName;
   readonly config: Config;
+  _isInitialized: boolean;
```

Comment:
can this be awaited? I think we had this as a method in the prototype

Reply:
What would be the use case for that? Anything that needs to await initialization should be putting tasks into `queue()`

Reply:
we used it in signInWithCredential https://github.com/firebase/firebase-js-sdk/blob/bc990a8a2800669b53e0c6f158947071930bbef6/packages/auth-ts/src/core/strategies/index.ts#L60

Reply:
Ah yeah so that isn't necessary -- that method then calls updateCurrentUser which tosses the update into the queue
https://github.com/firebase/firebase-js-sdk/blob/bc990a8a2800669b53e0c6f158947071930bbef6/packages/auth-ts/src/core/strategies/index.ts#L63